src/main/java/dev/openfga/sdk/api/client/ListStoresOptions.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * OpenFGA
+ * A high performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar.
+ *
+ * The version of the OpenAPI document: 0.1
+ * Contact: community@openfga.dev
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package dev.openfga.sdk.api.client;
+
+public class ListStoresOptions {
+    private Integer pageSize;
```

Comment:
Both these should be optional/nullable

Reply:
Yes, these are nullable. `Integer` is the object version of the primitive `int` that allows for null.

I'll write javadoc comments for the behavior

I could also update the return values of all the getters in these to Optional<Integer>, Optional<String>, etc. This comes at a little overhead in objects that need to be allocated and garbage collected. These are only intended to be used by the client, and the client also accepts null

I prefer not to add `@Nullable` annotations since adding/removing these can change the binary compatibility of the class

Reply:
The JSR 305 `javax.annotation.Nullable` annotations (with findbugs in the package name) have a much longer history https://jcp.org/en/jsr/detail?id=305 and [are used in 12k other projects](https://mvnrepository.com/artifact/com.google.code.findbugs/jsr305)

We already have a dependency on this coming from OpenAPI Generator's template and Jackson (used for serialization/deserialization)

JetBrains also has a @Nullable annotation and its package [is used in 5k other projects](https://mvnrepository.com/artifact/org.jetbrains/annotations)

Just as a note, every other project using some version of these packages is a dependency conflict that may have to be resolved, so using either of these is not 100% free.

---

Note 1: While working at AWS I saw some weird situations where introducing an annotation can be a breaking change. (Even if no other code or behavior changes.) I think these are unlikely situations to see without a very large user base and code that does wacky things with reflection.

Note 2: While IntelliJ is the most popular IDE these days, [nearly 40% of Java developers use something else](https://snyk.io/blog/intellij-idea-dominates-the-ide-market-with-62-adoption-among-jvm-developers/). Anecdotally, I've seen that IntelliJ is favored by younger engineers, and other IDEs are favored by more mature engineers.